### ISSUE TITLE
Default filter `allow_false:` named parameter not implemented

### ISSUE THREAD

**Issue by robin-parker** [dotliquid/dotliquid#438]
### Dotliquid version
2.1.457

### Expected behavior
Allow false to be returned by default filter when allow_false _named-parameter_ is set to true.
See: https://shopify.github.io/liquid/filters/default/#allowing-false

### Actual behavior
Not yet implemented.

### Steps to reproduce the Problem (you can add files)
```
     Helper.AssertTemplateResult(
                    expected: "\r\nfalse",
                    template: @"{% assign display_price = false %}
{{ display_price | default: true, allow_false: true }}"); // Since Liquid 5.0.0
```

**Comment by microalps** [comment]
Should this be a placeholder for all 5.0 features missing? I mentioned a few new tags in the wiki.

**Comment by robin-parker** [comment]
Makes sense to have a placeholder for 5.0.0 for tracking the feature gaps, the [Liquid change history](https://github.com/Shopify/liquid/blob/eab13a07d9861a38d993d2749ae25f06ff76426b/History.md) categorizes Shopify/liquid#1144 as a fix, rather than a new feature. 

`allow_false:` is the only case of a named parameter that I could find.

**Comment by MarioMajcicaAtABNAMRO** [comment]
Any progress on this?
